packages/mpl-widget-box.yaml

```diff
@@ -0,0 +1,5 @@
+repo: leejjoon/mpl_widget_box
+keywords: [widgets]
+section: interactivity
+description: Another gui-neutral widgets for matplotlib
```

Comment:
Can we have a more specific differentiator than 'Another'? Also, capitalize Matplotlib please.

Reply:
Or leave out "Another" if you don't find a meaningful qualification.

Reply:
Thanks for the suggestion. Would it be okay if I simply go with "GUI-neutral widgets for Matplotlib" then?

Reply:
I pushed a new description of "GUI-neutral widgets for Matplotlib w/ legend-like layout and more" with which I tried to give more specificity.